src/plugin.js

```diff
@@ -1,42 +1,72 @@
 import { createStore } from './store'
 
-// Vanilla Installer
+export const injectedPropertyName = 'stator'
 
-function lazyInject (Vue, key, setter) {
-  // Check if plugin not already installed
-  const installKey = `__stator_${key}_installed`
-  if (Vue[installKey]) {
-    return
-  }
+export function injectStator (Vue, options) {
+  lazyInject(Vue, injectedPropertyName, (vm) => {
+    return createStore(options, vm.$root)
+  })
 
-  Vue[installKey] = true
+  injectProperty(Vue, injectedPropertyName, '$state')
```

Comment:
It wasnt really useful to use the lazy injection for the stator properties. We can just inject them directly, we only want to lazy inject the store so the store is only created once really used